openlibrary/core/lending.py

```diff
@@ -95,6 +98,18 @@ def setup(config):
     except AttributeError:
         amazon_api = None
 
+def get_users_loan_history(ol_username):
```

Comment:
Given changes on archive.org, this block will no longer be necessary until OL has a privileged/protected API for loan history. Deprecating this block.

Reply:
resolved